### PR TITLE
research page tweaks

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: db/migrate_cards/data/cards/*css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    float:right;
+    border: 2px solid $border-color;
+    margin-left: 15px;
+}
+
+.SELF-un_sustainable_development_goal-topic_list .item-browse_item {
+    float: left;
+    width: 20%;
+    padding: 10px;
+}
```
Comment (xithan) [inline db/migrate_cards/data/cards/*css:15]: this removes css that is no longer necessary (according to Vignesh)

File: db/migrate_cards/data/cards/using_wiki_rate [hunk 2]
```diff
@@ -1,44 +1,31 @@
...
-          <a class="list-group-item" href="#Adding_Tagging_New_Sources">Adding & Tagging New Sources</a>
-          <a class="list-group-item" href="#Company_Profiles">Company Profiles</a>
-          <a class="list-group-item" href="#Join_a_Research_Group">Join a Research Group</a>
-          <a class="list-group-item" href="#Project_Research">Project Research</a>
-          <a class="list-group-item" href="#Exporting_data">Exporting Data</a>
-          <a class="list-group-item" href="#WikiRate_Support">WikiRate Support</a>
-          <a class="list-group-item" href="#Advanced_Features">Advanced Features</a>
-        </div>
-    </div>
+     {{_|panel_toc}}
```
Comment (ethn) [inline db/migrate_cards/data/cards/using_wiki_rate:28]: yes!

File: mod/wikirate_source/set/source_type/wikirate_link/download.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+end
+
+def file_size
+  @file_size ||= header[/.*Content-Length: (.*)\r\n/, 1].to_i
+end
+
+def max_size
+  # prevent from showing file too big while users are adding a link source
+  (max = Card["*upload max"]) ? max.db_content.to_i : 5
+end
```
Comment (ethn) [inline mod/wikirate_source/set/source_type/wikirate_link/download.rb:71]: safe to assume most of this is just moved without change?  Anything in particular warrant a closer look here?

**Review by ethn — APPROVED** [review]

File: mod/wikirate_source/set/source_type/wikirate_link/download.rb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+def after_download_success
+  file_field =
+    add_subfield :file, remote_file_url: file_url, type_id: FileID, content: "dummy"
+  file_field.director.catch_up_to_stage :validate
+  if file_field.errors.any?
+    remove_subfield :file
+    return
+  end
```
Comment (xithan) [inline mod/wikirate_source/set/source_type/wikirate_link/download.rb:17]: 13-17 is new, everything else is only reorganized

**Pull request merged**